examples/land-cover-map/SI_LULC_pipeline.ipynb

```diff
@@ -3407,21 +3403,21 @@
     "        self.predicted_scores_name = predicted_scores_name\n",
     "        \n",
     "    def execute(self, eopatch):\n",
-    "        ftrs = eopatch[self.features_feature[0]][self.features_feature[1]]\n",
+    "        features = eopatch[self.features_feature[0]][self.features_feature[1]]\n",
```

Comment:
This could actually be updated into `features = eopatch[self.features_feature]`. When this notebook was first written we didn't yet support such notation but now we do.